version/rawversion/version.go

```diff
@@ -7,4 +7,4 @@ package rawversion
 //
 // NOTE: remember to bump the version at the top of the top-level README.md
 // file when this is bumped.
-const RawVersion = "5.0.0-rc7"
+const RawVersion = "5.0.0-dev"
```

Comment:
Should this be 5.1.0-dev?

Reply:
nope, this is already on v5.0 branch

Reply:
but ya, we need to bump main to 5.1.0-dev

Reply:
https://github.com/containers/podman/pull/22054